src/transitions/Slide.js

```diff
@@ -82,14 +85,17 @@ export default class Slide extends Component {
   handleEntering = (element) => {
     const { transitions } = this.context.theme;
     element.style.transition = transitions.create('transform',
-      `${this.props.transitionDuration}ms`);
+      `${this.props.enterTransitionDuration}ms`, undefined, easing.easeOut);
```

Comment:
is `undefined` for `0ms`?
I'm wondering if we should change the API. That makes me think of https://twitter.com/brian_d_vaughn/status/828507525469253632.